# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xterm/reverse-video.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xterm-397/charproc.c.vanilla	2025-01-28 10:24:37.320039262 +0100
+++ xterm-397/charproc.c	2025-01-28 10:24:27.560039257 +0100
@@ -487,7 +487,7 @@
     Bres(XtNprinterNewLine, XtCPrinterNewLine, SPS.printer_newline, True),
     Bres(XtNquietGrab, XtCQuietGrab, screen.quiet_grab, False),
     Bres(XtNresizeByPixel, XtCResizeByPixel, misc.resizeByPixel, False),
-    Bres(XtNreverseVideo, XtCReverseVideo, misc.re_verse, False),
+    Bres(XtNreverseVideo, XtCReverseVideo, misc.re_verse, True),
     Bres(XtNreverseWrap, XtCReverseWrap, misc.reverseWrap, False),
     Bres(XtNscrollBar, XtCScrollBar, misc.scrollbar, False),
     Bres(XtNscrollKey, XtCScrollCond, screen.scrollkey, False),
